app/templates/feedback_sent.html

```diff
@@ -0,0 +1,16 @@
+{% extends theme('layouts/_onecol.html') %}
+
+{% block page_title %}{{_("Thankyou for your feedback")}}{% endblock %}
+
+{% block main %}
+
+<div class="u-mb-s">
+    <h1 class="saturn">Feedback Sent</h1>
+    <p>Thank you for your feedback.</p>
+</div>
+
+<div class="u-mt-s">
+    <p class="mars">It is now safe to close this window.</p>
```

Comment:
Had this reviewed for content and flagged with BenA, looks like it's going to stay as-is.